docs/update.md

```diff
@@ -0,0 +1,42 @@
+# Introduction
+
+This document describes how to update an Aeternity node using a release binary:
+
+* [Manual update](#manual-update);
+
+## Manual update
+
+### Ubuntu 18.04
+
+Be sure to have a backup of your files, don't remove your node database.
+
+Download the latest version of æternity node (Iris is 6.0.0)
```

Comment:
The latest version of Iris is `6.1.0`, but perhaps we shouldn't hard-code the latest version here. "(Iris is 6.x.y)", perhaps, or simply "Download the latest version of the aeternity node".